### DISCUSSION TITLE
Model location

### DISCUSSION THREAD

**Discussion by iplayfast** [open-webui/open-webui#6097] [General]
# Feature Request
I'm using open-webui a lot, but sometimes I need to access my computer through ssh. (ie no browser available). I'd like to run ollama with the model that has been created in open-webui, but the models that are created with open-webui don't seem to be around. (which is very confusing).

I installed with the docker option, where ollama is already on my system. I assumed that the models would be wherever ollama put them.

But I created a new model that has lots of extra knowledge, but I can't access it from ollama. 

Thanks for a wonderful project!

**Comment by tjbck** [comment]
Open WebUI Models are separate from Ollama models to ensure maximum compatibility with other LLM providers, you should look into: https://docs.openwebui.com/api/
